service/src/main/java/com/epam/esm/dto/entityDTO/RoleDTO.java

```diff
@@ -11,6 +12,6 @@
 @RequiredArgsConstructor
 @EqualsAndHashCode(callSuper = true)
 public class RoleDTO extends EntityDTO {
-    //todo: custom validator with enum
+    @EnumNamePattern(regex = "USER|ADMIN")
```

Comment:
Please use `@Pattern(regexp = "yor regex")` from `javax.validation.constraints.Pattern` and remove EnumNamePattern and EnumNamePatternValidator classes

Reply:
fixed it